ACHNBrowserUI/ACHNBrowserUI/views/customTasks/CustomTasksListView.swift

```diff
@@ -0,0 +1,84 @@
+//
+//  CustomTasksListView.swift
+//  ACHNBrowserUI
+//
+//  Created by Jan on 24.05.20.
+//  Copyright © 2020 Thomas Ricouard. All rights reserved.
+//
+
+import SwiftUI
+import SwiftUIKit
+import Backend
+
+struct CustomTasksListView: View {
+    @EnvironmentObject private var collection: UserCollection
+    @Environment(\.presentationMode) private var presentationMode
+    @State private var editMode = EditMode.inactive
+    
+    private var closeButton: some View {
+        Button(action: {
+            self.presentationMode.wrappedValue.dismiss()
+        }, label: {
+            Image(systemName: "xmark.circle.fill")
+                .style(appStyle: .barButton)
+                .foregroundColor(.acText)
+        })
+        .buttonStyle(BorderedBarButtonStyle())
+        .accentColor(Color.acText.opacity(0.2))
+        .safeHoverEffectBarItem(position: .leading)
+    }
+    
+    private var orderButton: some View {
+        Button(action: {
+            if (self.editMode == EditMode.active) {
+                self.editMode = EditMode.inactive
+            }
+            else {
+                self.editMode = EditMode.active
+            }
```

Comment:
I would rather add a `toggle()` function in an `EditMode` extension somewhere.
It's working, for sure, but all this `if/else` code seems a bit overkill :D.

I would like to see something like this:
```swift
// EditMode extension somewhere in our extension folder
extension EditMode {
    mutating func toggle() {
        switch self {
        case .active: self = .inactive
        case .inactive: self = .active
        case .transient: break
        @unknown default: break
        }
    }
}
```
```swift
Button(action: { editMode.toggle() }, label: { ... })
```

Reply:
I love this one! Used it!